resources/install.rb

```diff
@@ -113,8 +114,10 @@
       mode '0644'
       source lazy { node.run_state['haproxy']['conf_template_source'][config_file] }
       cookbook lazy { node.run_state['haproxy']['conf_cookbook'][config_file] }
-      notifies :enable, 'poise_service[haproxy]', :immediately
-      notifies :restart, 'poise_service[haproxy]', :delayed
+      unless install_only
+        notifies :enable, 'poise_service[haproxy]', :immediately
```

Comment:
The problem that I encounter with the systemd shim is that I'm using haproxy in front of live streaming ingestion. The restart is terminating the connections. haproxy does have a method of restarting that does not terminate existing connections. This works for me, so I've made recipes for restarting and reloading.

That said, the recipes in this repo are very helpful for install / config generation and I'd like to continue using them for that.